2021-06-18 04:00:05,429 INFO    MainThread:4028368 [internal.py:wandb_internal():91] W&B internal server running at pid: 4028368, started at: 2021-06-18 04:00:05.428556
2021-06-18 04:00:05,435 INFO    WriterThread:4028368 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_040003-3ouwwhm2/run-3ouwwhm2.wandb
2021-06-18 04:00:05,437 DEBUG   HandlerThread:4028368 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 04:00:07,672 DEBUG   HandlerThread:4028368 [meta.py:__init__():39] meta init
2021-06-18 04:00:07,672 DEBUG   HandlerThread:4028368 [meta.py:__init__():53] meta init done
2021-06-18 04:00:07,672 DEBUG   HandlerThread:4028368 [meta.py:probe():210] probe
2021-06-18 04:00:07,680 DEBUG   HandlerThread:4028368 [git.py:repo():33] git repository is invalid
2021-06-18 04:00:07,681 DEBUG   HandlerThread:4028368 [meta.py:_save_pip():57] save pip
2021-06-18 04:00:07,683 DEBUG   HandlerThread:4028368 [meta.py:_save_pip():71] save pip done
2021-06-18 04:00:07,684 DEBUG   HandlerThread:4028368 [meta.py:_save_conda():78] save conda
2021-06-18 04:00:11,742 DEBUG   HandlerThread:4028368 [meta.py:_save_conda():86] save conda done
2021-06-18 04:00:11,743 DEBUG   HandlerThread:4028368 [meta.py:probe():252] probe done
